pygmt/src/meca.py

```diff
@@ -221,6 +221,33 @@ def meca(  # noqa: PLR0912, PLR0913, PLR0915
     r"""
     Plot focal mechanisms.
 
+    Different conventions are supported:
+
+    .. list-table::
```

Comment:
Maybe add a table title?
```suggestion
    .. list-table:: Supported focal mechanism conventions
```